cxx-sensors/src/main/java/org/sonar/cxx/sensors/utils/CxxIssuesReportSensor.java

```diff
@@ -267,12 +268,26 @@ private void saveViolation(SensorContext sensorContext, CxxReportIssue issue) {
       }
     }
 
+    for (CxxReportLocation location : issue.getFlow()) {
+      NewIssueLocation newIssueLocation = createNewIssueLocationFile(sensorContext, newIssue, location, affectedFiles);
+      if (newIssueLocation != null) {
+        newIssueFlow.add(newIssueLocation);
+      } else {
+        LOG.error("Failed to create new issue location from flow location {}", location);
+        newIssueFlow.clear();
+        break;
+      }
+    }
+
     if (!newIssueLocations.isEmpty()) {
       try {
         newIssue.at(newIssueLocations.get(0));
         for (int i = 1; i < newIssueLocations.size(); i++) {
           newIssue.addLocation(newIssueLocations.get(i));
         }
+        if (newIssueFlow.size() > 1) {

```

Comment:
Please add also text from unit test here to explain why > 1

Reply:
Done